Move manual backup docs (#203)
nebari-dev/nebari-docs: docs/docs/how-tos/manual-backup.md

First, you might think you can just make a new `pod.yaml` file, this time specifying `claimName: "hub-db-dir"` instead of `claimName: "jupyterhub-dev-share"`. However, `hub-db-dir`
is 'Read Write Once' - the 'Once' meaning it can only be mounted to one pod at a time, but the JupyterHub pod will already have this mounted! So the same approach will not work
here.

Instead of mounting to a new 'debugger pod' you have to access the JupyterHub pod directly using the `kubectl` CLI.

Look up the JupyterHub pod:

```bash
kubectl get pods -n dev
```

It will be something like `hub-765c9488d6-8z4nj`.

Get a shell into that pod:

```bash
kubectl exec -n dev --stdin --tty hub-765c9488d6-8z4nj -- /bin/bash
```

There is no need to TAR anything up since the only file required to be backed up is `/srv/jupyterhub/jupyterhub.sqlite`.

### Backing up JupyterHub DB

Now we just need to upload the file to S3. You might want to [install the AWS CLI tool](#installations) as we did before, however, as the Hub container is a rather restricted
environment the recommended approach is to upload files to AWS S3 buckets using curl.

> For more details please refer to the [using curl to access AWS S3 buckets](./awss3curl.md) documentation.
> 

Reviewer: ```suggestion
For more details please refer to the [using curl to access AWS S3 buckets] documentation.

```

### Restoring JupyterHub DB

You will need to overwrite the file `/srv/jupyterhub/jupyterhub.sqlite` based on the version backed up to S3.

You should restart the pod:

```bash
kubectl delete -n dev pod hub-765c9488d6-8z4nj
```

> As for uploads, [you may need to use curl to download items from an AWS S3 bucket](./awss3curl.md).
> 

Reviewer: ```suggestion
As for uploads, [you may need to use curl to download items from an AWS S3 bucket]

```

## Keycloak user/group database

Nebari provides a simple script to export the important user/group database. Your new Nebari cluster will recreate a lot of Keycloak config (including new Keycloak clients which will
have new secrets), so only the high-level Group and User info is exported.

If you have a heavily customized Keycloak configuration, some details may be omitted in this export.

### Export Keycloak

> The export script is at [`nebari/scripts/keycloak-export.py`](https://github.com/Quansight/qhub/blob/main/scripts/keycloak-export.py).
> 

Reviewer: ```suggestion
The export script is at [`nebari/scripts/keycloak-export.py`](https://github.com/nebari-dev/nebari/blob/main/scripts/keycloak-export.py).

```

Locate your `nebari-config.yaml` file, for example by checking out of your Git repo for you Nebari. Activate a virtual environment with the `nebari` Python package installed.

This assumes that the password visible in the `nebari-config.yaml` file under the `security.keycloak.initial_root_password` field is still valid for the root user.

If not, first set the `KEYCLOAK_ADMIN_PASSWORD` environment variable to the new value.

Run the following to create the export file:

```shell
python nebari/scripts/keycloak-export.py -c nebari-config.yaml > exported-keycloak.json
```

You may wish to upload the Keycloak export to the same S3 location where you uploaded the TAR file in the NFS section.

### Import Keycloak

> To re-import your users and groups, [login to the /auth/ URL](../installation/login.md) using the root username and password.
> 

Reviewer: ```suggestion
To re-import your users and groups, [login to the /auth/ URL] using the root username and password.

```
